addons/website_slides/models/res_partner.py

```diff
@@ -57,7 +57,6 @@ def _compute_slide_channel_company_count(self):
                 partner.slide_channel_company_count = 0
 
     def action_view_courses(self):
-        action = self.env["ir.actions.actions"]._for_xml_id("website_slides.slide_channel_action_overview")
-        action['name'] = _('Followed Courses')
-        action['domain'] = ['|', ('partner_ids', 'in', self.ids), ('partner_ids', 'in', self.child_ids.ids)]
+        action = self.env["ir.actions.actions"]._for_xml_id("website_slides.slide_channel_partner_action")
+        action['context'] = {'search_default_partner_id': self.ids}
```

Comment:
This breaks the feature if you come from a company record.
Example:
- on your run build
- on the contact form of "Marc Demo"
- create "YourCompany" (using the small "+ Create company" button)
- open the created company record
- use the "Courses" stat button to reach the courses
- -> doesn't show any courses

I think you have to do this instead:
```py
self.ensure_one()
action = self.env["ir.actions.actions"]._for_xml_id("website_slides.slide_channel_partner_action")
action['name'] = _('Followed Courses')
if self.is_company:
    action['domain'] = [('partner_ids', 'in', self.child_ids.ids)]
else:
    action['context'] = {'search_default_partner_id': self.id}
```

And I would also keep the name "Followed Courses" otherwise it shows "Attendees" in the breadcrumb, which doesn't really make sense.